src/qdirfiletree.h

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright (C) MO2 Team. All rights reserved.
+
+This file is part of Mod Organizer.
+
+Mod Organizer is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Mod Organizer is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Mod Organizer.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef ARCHIVEFILENETRY_H
+#define ARCHIVEFILENTRY_H
+
+#include <QDir>
+
+#include "ifiletree.h"
+
+
+/**
+ * Class that expose a directory on the drive, using QDir, as a `MOBase::IFileTree`.
```

Comment:
Maybe mention that it's lazily populated

Reply:
I can add a comment here, but it's specified in `IFileTree` I think (all `IFileTree` are lazily populated).

Reply:
Ah I thought it was implementation specific

Reply:
Nope, `IFileTree` will call `doPopulate` when needed.